src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -1,121 +1,128 @@
+    <img
+        title="Guided install from the UI"
+        alt="Guided install from the UI"
+        src="/images/kubernetes_screenshot-crop_gather-log-data.webp"
+    />
 
-     ```yml
-     newrelic-logging:
-       enabled: true
-       endpoint: https://log-api.eu.newrelic.com/log/v1
-     ```
-   </Callout>
+    <figcaption>
+        Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/) > Integrations & Agents**</DNT> and click the <DNT>**Guided install**</DNT> tile. Then select <DNT>**Kubernetes**</DNT> and <DNT>**Guided**</DNT>.
+    </figcaption>
 
-3. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
+    <Callout variant="important">
 
-<InstallFeedback/>
+    If you're [using a Kubernetes secret](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L8-L25) to store the New Relic <InlinePopover type="licenseKey" />, the `newrelic-logging` chart defaults to sending logs to the US API endpoint. If the license key belongs to an EU or FedRAMP account, and a secret is used for key storage, you must update the endpoint setting with the appropriate value from the [API reference docs](/docs/logs/log-api/introduction-log-api/#endpoint). Here's an example of how to set this for EU accounts:
 
-## Install Fluent Bit output plugin [#fluent-bit]
+    ```yml
+        newrelic-logging:
+            enabled: true
+            endpoint: https://log-api.eu.newrelic.com/log/v1
+    ```
+    </Callout>
 
-New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
+3. Generate some traffic and wait a few minutes, then [check your account](#find-data) for data.
 
-You can install it in your cluster using our Helm chart in two ways.
+<InstallFeedback />
 
-### Use our guided install [#helm-guided-install]
+## Fluent Bit output plugin [#fluent-bit]
 
-Although the [newrelic-logging](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging) chart works as a standalone, we recommend installing it as part of the [nri-bundle](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart.
+New Relic has a [Fluent Bit](https://fluentbit.io/) [output plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to forward your logs to New Relic log management. This plugin is also provided in a standalone Docker image that can be installed in a Kubernetes cluster in the form of a DaemonSet, also known as the Kubernetes plugin.
```

Comment:
@nbaenam I thought that the `DaemonSet` from New Relic was the Kubernetes plugin, which seemed to be a subset of the overall Kubernetes integration. Is that true? If so, are we just saying you could have two DaemonSets: one for monitoring pods (the Kubernetes plugin) and one for forwarding logs (the Fluent Bit image)? I was confused by the "...also known as the Kubernetes plugin."

Reply:
@mangulonr, what do you think?